sysutils/calcurse/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
 
 name            calcurse
-version         4.5.1
+version         4.8.1
 categories      sysutils
 platforms       darwin
```

Comment:
`platforms darwin` is now implicit, and this line can be removed from Portfiles.